R/constructNWISURL.r

```diff
@@ -305,3 +305,27 @@ constructWQPURL <- function(siteNumber,parameterCd,startDate,endDate,zip=FALSE){
   return(url)
 
 }
+
+#' Construct URL for NWIS water use data service
+#' 
+#' 
+
+constructUseURL <- function(years,stateAB,county){ 
```

Comment:
I'm unsure about using stateCd as a variable name, when I need to call my stateCdLookup function that uses the stateCd data frame.  I've gotten errors with it trying to look up things in the variable, but I can't seem to reproduce it consistently.